www/pfSense-pkg-vHosts/files/usr/local/www/vhosts_certs.php

```diff
@@ -0,0 +1,465 @@
+<?php
```

Comment:
I am really not keen on what this page does. The Certificate Manager already serves this purpose, giving someone another way to import is unnecessary. If the Certificate Manager does not support something this needs, we should fix the Certificate Manager instead of reimplementing it.